src/main/java/ch/njol/skript/lang/globals/GlobalOptions.java

```diff
@@ -0,0 +1,34 @@
+package ch.njol.skript.lang.globals;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.config.Config;
+import ch.njol.skript.lang.OptionRegistry;
+
+import java.io.IOException;
+import java.util.*;
+
+/**
+ * Represents the 'globals/options.sk' file
+ */
+public class GlobalOptions extends GlobalFile {
+
+	public GlobalOptions() {
+		super(Skript.getAddonInstance(), "options");
```

Comment:
Should use `Skript.instance()`, or even better, just have the constructor take in a `SkriptAddon` instance.